pkg/proto/jmespath/jmespath.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
```

Comment:
I think this ought to live at pkg/proto/configuration/jmespath, as it only pertains to the configuration aspect of Buildbarn binaries. We never intend to send these messages "across the wire".